docs/integrations/ceramicExample.md

```diff
@@ -0,0 +1,219 @@
+---
+sidebar_position: 3
+---
+
+# ComposeDB (Ceramic)
```

Comment:
Ideally we have a table here with objectives for what this page will cover! 

In some docs, it's in the format:

```
## Objectives
At completion of this reading you should be able to:

- Explain at a high level what encryption and programmatic signing is.
```